viz/cmd/testdata/tap_busy_output_wide.golden

```diff
@@ -1,2 +1,2 @@
-req id=1:0 proxy=out src=0.0.0.1:0 dst=[ff01::1]:0 tls=true :method=GET :authority=localhost :path=/some/path dst_res=po/my-pod
-end id=1:0 proxy=out src=0.0.0.1:0 dst=[ff01::1]:0 tls= grpc-status=Code(666) duration=0µs response-length=1337B
+req id=1:0 proxy=out src=0.0.0.1:0 dst=[ff01::1]:0 tls=true :method=GET :authority=localhost :path=/some/path  dst_pod=my-pod dst_tls=true 
```

Comment:
it looks like there's an extra space before `dst_pod` that wasn't there previously?